neuraxle/pipeline.py

```diff
@@ -316,65 +299,58 @@ def should_resume(self, data_container: DataContainer, context: ExecutionContext
         return False
 
 
-class CustomPipelineMixin:
-    """
-    Boilerplate code for custom pipelines that only implements handle methods.
-
-    .. seealso::
-        :class:`~neuraxle.pipeline.MiniBatchSequentialPipeline`,
-        :class:`~neuraxle.api.DeepLearningPipeline`
-    """
-    def transform(self, data_inputs: Any):
-        """
-        :param data_inputs: the data input to transform
-        :return: transformed data inputs
+class CustomHandlerMethodsMixin:
```

Comment:
I have no idea what is this used for. Also, there is no docstring.

Reply:
it was a duplicate of _CustomHandlerMethods. removed.